be/src/exec/pipeline/exchange/local_exchange_source_operator.cpp

```diff
@@ -41,6 +41,10 @@ Status LocalExchangeSourceOperator::add_chunk(ChunkPtr chunk, const std::shared_
     if (_is_finished) {
         return Status::OK();
     }
+
+    // unpack chunk's const column, since Chunk#append_selective cannot be const column
+    chunk->unpack_and_duplicate_const_columns();
```

Comment:
if other case return const_null, it will crash in here too...

Reply:
Yep. Later we may support const column in Chunk/Column's `append_selective` to fix this totally.